src/javascript/binary/static/verify_email.js

```diff
@@ -0,0 +1,30 @@
+//get gtm layer from ashkan before release
```

Comment:
you can add // todo:
to comments like this, and then grep them before committing ;)